### ISSUE TITLE
Multi-sample dataset ---> Reference panel, Train set, Test set

### ISSUE THREAD

**Issue by Me-AU** [Me-AU/SALAI-Net#1]
Create scripts to divide a multi-sample dataset into:
- A reference panel
- A training set
- A test set
